### ISSUE TITLE
typeClass not applying styles

### ISSUE THREAD

**Issue by rwwagner90** [sir-dunxalot/ember-tooltips#58]
I am doing the following:

```js
renderTooltip(element, {
      content,
      effectClass: 'grow',
      hideOn: 'click',
      place: 'top',
      showOn: 'click',
      typeClass: 'light'
    });
```

However, the tooltip remains the dark 'none' style. Am I doing something wrong? I noticed that the darsain library seems to use classes like `light` and `success` but it appears this addon is adding `tooltip-` in front of the classes maybe?

I can't actually look at the markup for the tooltip because I can't figure out how to keep them open with dev tools open.

**Comment by sir-dunxalot** [comment]
I'll dig into this issue ASAP. It could be related to #49.

As for debugging, try using a tooltip event like `ready` or `click`. Maybe that will help?

**Comment by sir-dunxalot** [comment]
This has been fixed with release `1.0.0`: https://github.com/sir-dunxalot/ember-tooltips/releases/tag/1.0.0

`typeClass` has been removed. Instead, you can just pass a regular class to the tooltip:

```hbs
{{tooltip-on-component class='tooltip-light'}}
```

**Comment by rwwagner90** [comment]
Awesome! Thanks @sir-dunxalot
